scripts/makefiles/Protobuf.mk

```diff
@@ -135,7 +147,7 @@ API_V3_PATCHED=$(API_V3_PATCHED_DIR)/query_service.proto
 patch-api-v3:
 	mkdir -p $(API_V3_PATCHED_DIR)
 	cat idl/proto/api_v3/query_service.proto | \
-		$(SED) -f ./proto-gen/patch-api-v3.sed \
+		$(SED) -f ./proto-gen/patch.sed \
```

Comment:
I didn't realize the sed file is reusable.

Reply:
yeah its just a generic file that adds annotations for timestamps and durations 
```sed
s|import "google/protobuf/duration.proto";|import "google/protobuf/duration.proto";\
\
import "gogoproto/gogo.proto";\
\
option (gogoproto.marshaler_all) = true;\
option (gogoproto.unmarshaler_all) = true;\
option (gogoproto.sizer_all) = true;\
|g

s|google.protobuf.Timestamp \(.*\);|google.protobuf.Timestamp \1 \
  [\
  (gogoproto.nullable) = false,\
  (gogoproto.stdtime) = true\
  ];|g

s|google.protobuf.Duration \(.*\);|google.protobuf.Duration \1 \
  [\
  (gogoproto.nullable) = false,\
  (gogoproto.stdduration) = true\
  ];|g

```